src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -742,4 +773,44 @@ public void receiveSimpleMessageFromPartitionAndOffset(TestContext context) {
 
         deleteAsync.await();
     }
+
+    //@Test
+    public void sendToOnePartitionTest(TestContext context) {
```

Comment:
is it disabled on purpose?

Reply:
Yeah. From the API spec I see `POST /topics/(string: topic_name)/partitions/(int: partition_id)` request should do `"Produce messages to one partition of the topic"` but in our bridge, it is fully ignored (AFAICS). So this is the test which we can allow after this is implemented.

Reply:
But using that endpoint is another way to do that. You can already send to one partition when you specify "partition" field in a record in the "records" collection when you do a POST on `/topics/topic_name`

Reply:
Ah! I think I got it. You added test but not implementing the feature yet I guess. Well, I would avoid that, it creates just confusion imho. This PR is about producer error handling not implementing that endpoint.

Reply:
Ok, I just wanted to write a test which uses this endpoint. And I did O:) However, it does not cope properly with the situation which is described as "If no partition is provided, one will be chosen based on the hash of the key. If no key is provided, the partition will be chosen for each message in a round-robin fashion."

Reply:
That description works with the generic `/topics/topic-name` endpoint of course, not this one.